infra/main.bicep

```diff
@@ -201,14 +205,17 @@ module backend 'core/host/appservice.bicep' = {
     serverAppId: serverAppId
     clientSecretSettingName: !empty(clientAppSecret) ? 'AZURE_CLIENT_APP_SECRET' : ''
     authenticationIssuerUri: authenticationIssuerUri
+    use32BitWorkerProcess: appServiceSkuName == 'F1'
```

Comment:
@jongio Wondering if I should change core/appservice to always set these based on the SKU?